test/gnosisSafePersonalEdition.js

```diff
@@ -4,6 +4,10 @@ const solc = require('solc')
 
 const GnosisSafe = artifacts.require("./GnosisSafe.sol")
 const ProxyFactory = artifacts.require("./ProxyFactory.sol")
+// const MockContract = artifacts.require('@gnosis.pm/mock-contract/MockContract');
```

Comment:
You can use the mock via npm now. Add `
    "mock-contract": "github:fleupold/mock-contract#82b0aba9e57d20f0aa9c22b125a3f60a1329262c",` to the `devDependencies`

also in you mock Token.sol add `
import "mock-contract/contracts/MockContract.sol";`. This will tell solc to compile the MockContract.sol and you can use it just by `artifacts.require('./MockContract.sol');`

Reply:
Yes. Now I am using the mock-contract through npm